src/tests/JIT/Regression/JitBlue/Runtime_106124/Runtime_106124.cs

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
```

Comment:
I assume this is a Fuzzlyn found example? Can you add the Fuzzlyn header?

Reply:
Well, yes and no 🙂 . I was trying to minimise [this](https://github.com/dotnet/runtime/issues/106124#issue-2455638331) example when I encountered this issue. The original example doesn't have this pattern so didn't add it in the header. Happy to change to whatever you suggest 👍

Reply:
Do you mean while trying to minimize #105474? I did hit this assert during Fuzzlyn runs as well, but I assumed it was the same underlying problem, so I didn't open an issue about it.
Fine to leave it without, I just noticed the variable names looked Fuzzlyn generated, and we like to be able to track which issues are fuzzer found.

Reply:
Sorry, I meant - minimising #105720.